src/db2ia/db2ia.cc

```diff
@@ -48,6 +48,12 @@ Napi::Object InitAll(Napi::Env env, Napi::Object exports)
                                 SQL_ATTR_SERVER_MODE, // SQLINTEGER Attribute - Enable Server Mode by default
                                 &param,               // SQLPOINTER Value Appropriate value for Attribute
                                 0);                   //SQLINTEGER StringLength of Value in bytes if the attribute value is a character string
+  
+  // SQL_ATTR_NON_HEXCCSID will change the job CCSID to the default job CCSID if
+  // the the job CCSID is 65535. The default job CCSID is guaranteed to be a
+  // non-65535 CCSID. This would help us avoid the common error:
+  // "Character conversion between CCSID 1208 and CCSID 65535 not valid"
+  sqlReturnCode = SQLSetEnvAttr(envh, SQL_ATTR_NON_HEXCCSID, &param, 0);
```

Comment:
I ran into issue compiling on stating `SQL_ATTR_NON_HEXCCSID' was not declared in this scope`.

```
make: Entering directory '/home/amusse/nodejs-idb-connector/build'
  CXX(target) Release/obj.target/db2ia/src/db2ia/db2ia.o
../src/db2ia/db2ia.cc: In function 'Napi::Object InitAll(Napi::Env, Napi::Object)':
../src/db2ia/db2ia.cc:56:39: error: 'SQL_ATTR_NON_HEXCCSID' was not declared in this scope
   sqlReturnCode = SQLSetEnvAttr(envh, SQL_ATTR_NON_HEXCCSID, &param, 0);
                                       ^~~~~~~~~~~~~~~~~~~~~
```

I searched `/usr/include/sqlcli.h` and sure enough it was not defined in that file.

@kadler Is `SQL_ATTR_NON_HEXCCSID` defined somewhere else?

Reply:
I noticed with python [ibm_db2](https://github.com/kadler/python-ibmdb/blob/9b2b018afbba308b9a343809683f784875d24c70/IBM_DB/ibm_db/ibm_db.c#L129-L131) connector there is a check for it:


```
#ifndef SQL_ATTR_NON_HEXCCSID
#define SQL_ATTR_NON_HEXCCSID 10203
#endif
```

I just find it weird that its [documented here](https://www.ibm.com/support/knowledgecenter/ssw_ibm_i_72/cli/rzadpfnsenva.htm) but not defined in the header file?

Reply:
I think the header file missed getting updated or it is only in a newer IBM i release.